sample_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -168,6 +178,16 @@ private void DoExitGame()
 	public void CheckAPIKey(string value)
 	{
 		apiKey = value;
+
+		if (apiKey == "")
+		{
+			apiKeyInputField.text = "Enter your API key here...";
+		}
+		else
+		{
+            apiKeyInputField.text = apiKey;
+        }
```

Comment:
Formatting here looks incorrect.

Reply:
That's weird because it doesn't look like that in VS